packages/workspace/src/workspace/workspace.ts

```diff
@@ -408,6 +410,29 @@ export const loadWorkspace = async (config: WorkspaceConfigSource, credentials:
       })()
       return { serviceName, status, date }
     },
+    getValue: async (id: ElemID): Promise<Value | undefined> => {
+      const topLevelID = id.createTopLevelParentID().parent
+
+      const workspaceElement = await naclFilesSource.get(topLevelID)
+      const stateElement = await state().get(topLevelID)
+
+      if (workspaceElement === undefined && stateElement === undefined) {
+        log.error(`ElemID not found ${id}`)
```

Comment:
nit:
```suggestion
        log.error('ElemID not found %s', id.getFullName())
```
also, I don't think this deserves to be an error in the log, we don't know all the scenarios this might be called, it might be a valid use case to try to get an element that doesn't exist.
errors in the log have an impact on OSS users, we don't want to flood the log false errors.
I'd make this a debug log at most